notebooks/02-ingestion/04-table-datatypes.ipynb

```diff
@@ -0,0 +1,851 @@
+    "SELECT\n",
+    "  TIME_PARSE(\"depaturetime\") AS \"__time\",\n",
+    "  \"Year\",\n",
+    "  \"Reporting_Airline\",\n",
+    "  \"Origin\",\n",
+    "  \"Dest\",\n",
+    "  \"Distance\"\n",
+    "FROM \"ext\"\n",
+    "WHERE \"depaturetime\" <> 0\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "\n",
+    "display.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-flights-types-1')\n",
+    "display.table('example-flights-types-1')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "889be0da-36ec-4510-876a-c7c040355b37",
+   "metadata": {},
+   "source": [
+    "You can query the `INFORMATION_SCHEMA` schema's `COLUMNS` table to [get information](https://druid.apache.org/docs/latest/querying/sql-metadata-tables#columns-table) for any TABLE in Druid.\n",
```

Comment:
Isn't this the same as the `display.table('example-flights-types-1`)` above? I think the explanation in the next cell is still valid. Doing it again directly in the INFORMATION_SCHEMA seems superfluous to me.

Reply:
Ahaaa yes you're right. Removing the display.table('example-flights-types-1')